examples/kotlin/src/main/kotlin/org/androidannotations/gradle/activity/HelloAndroidActivity.kt

```diff
@@ -12,7 +12,7 @@ import org.androidannotations.gradle.R
 import java.util.Date
 
 @EActivity(R.layout.main)
-open public class HelloAndroidActivity : Activity() {
+open class HelloAndroidActivity : Activity() {
```

Comment:
Classes with no access modifier are always public in kotlin. There is no need to type public.